src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,49 @@
+package core.basesyntax;
+
+import core.basesyntax.model.FruitRecord;
+import core.basesyntax.service.parse.DataParser;
+import core.basesyntax.service.parse.DataParserImpl;
+import core.basesyntax.service.read.FileReader;
+import core.basesyntax.service.read.FileReaderImpl;
+import core.basesyntax.service.report.ReportService;
+import core.basesyntax.service.report.ReportServiceImpl;
+import core.basesyntax.service.strategy.TypeStrategy;
+import core.basesyntax.service.strategy.TypeStrategyImpl;
+import core.basesyntax.service.strategy.strategyimpl.BalanceOperation;
+import core.basesyntax.service.strategy.strategyimpl.OperationHandler;
+import core.basesyntax.service.strategy.strategyimpl.PurchaseOperation;
+import core.basesyntax.service.strategy.strategyimpl.ReturnOperation;
+import core.basesyntax.service.strategy.strategyimpl.SupplyOperation;
+import core.basesyntax.service.write.FileWriter;
+import core.basesyntax.service.write.FileWriterImpl;
+import java.util.HashMap;
+import java.util.List;
+
+public class Main {
+    private static final String PATH_TO_READ = "src/main/resources/test.csv";
+    private static final String PATH_TO_WRITE = "src/main/resources/report.csv";
```

Comment:
Avoid hardcoding file paths in your service classes. It's better to pass these as parameters to the `main` method or read them from a configuration file.